test/common/debugger.js

```diff
@@ -7,7 +7,13 @@ const BREAK_MESSAGE = new RegExp('(?:' + [
   'exception', 'other', 'promiseRejection',
 ].join('|') + ') in', 'i');
 
-const TIMEOUT = common.platformTimeout(5000);
+let TIMEOUT = common.platformTimeout(5000);
+if (common.isWindows) {
+  // Some of the windows machines in the CI need more time to receive
+  // the outputs from the client.
+  // https://github.com/nodejs/build/issues/3014
+  TIMEOUT = common.platformTimeout(15000);
```

Comment:
I am not sure if there is a more specific way to test that we are on the azure win10 vs2019 machiens, but I believe a longer timeout isn't going to be a problem for Windows machines anyway, so it's probably ok to just increase the timeout for all Windows machines (if they really take too long they are going to be terminated by the test runner anyway)